lib/components/Signature.js

```diff
@@ -1,12 +1,24 @@
 //@flow
 import React from 'react';
-import { GLView } from 'expo-gl';
-import { PanResponder, PixelRatio } from 'react-native';
-import { vec2 } from 'gl-matrix';
+import {
+  GLView
```

Comment:
What benefit is there to splitting collections that fit very comfortably on a single line?